tests/Fixtures/Preload/Case020_Out.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+use App\Foo;
+
+class_exists('App\Foo');
+class_exists(Foo::class);
```

Comment:
This is not a nice solution. It will duplicate the `class_exists(Foo::class);` for the `App\Foo` class. However, it is soo much more easier. 

Allowing this would avoid parsing use statements and class alias to figure out that `App\Foo` is already "preloaded"

Reply:
> (...) and class alias (...)

I don't see class alias here in this test case, but if you need to handle them, maybe use https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/2.16/src/Tokenizer/Analyzer/NamespaceUsesAnalyzer.php ?